backend/src/routes/classroom.ts

```diff
@@ -1,12 +1,62 @@
-import { Request, Response, Router } from 'express';
+import { NextFunction, Request, Response, Router } from 'express';
 import Classroom from 'models/classroom';
+import { makeError } from 'helpers/error';
+import { NOT_FOUND } from 'http-status';
 
 const router = Router();
 
-router.get('/', async (req: Request, res: Response) => {
-  const classrooms = await Classroom.findAll();
+router.get('/', async (req: Request, res: Response, next: NextFunction) => {
+  await Classroom.findAll()
```

Comment:
Why the awaits in all of these route handlers?

Reply:
To be honest, I was on autopilot. I followed first prototype and didn't think about it. 😅